src/code.cloudfoundry.org/loggregator/plumbing/tls.go

```diff
@@ -142,16 +166,26 @@ func addCA(tlsConfig *tls.Config, tlsCert tls.Certificate, caCertFile string) er
 	return nil
 }
 
-func NewCredentials(certFile, keyFile, caCertFile, serverName string) (credentials.TransportCredentials, error) {
-	tlsConfig, err := NewMutualTLSConfig(
-		certFile,
-		keyFile,
-		caCertFile,
-		serverName,
-	)
+func newMutualTLSConfig(certFile, keyFile, caCertFile, serverName string) (*tls.Config, error) {
+	tlsCert, err := tls.LoadX509KeyPair(certFile, keyFile)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed to load keypair: %s", err.Error())
 	}
 
-	return credentials.NewTLS(tlsConfig), nil
+	tlsConfig := &tls.Config{
```

Comment:
`NewTLSConfig()` has dropped it's MinVersion. Therefore, we want a stricter requirement.